pkg/kafka/response.go

```diff
@@ -41,6 +42,25 @@ func (res *ResponseMessage) String() string {
 	return string(b)
 }
 
+// ReadResponse will read a Kafka response from an io.Reader and return the
+// message or an error.
+func ReadResponse(reader io.Reader) (*ResponseMessage, error) {
+	rsp := &ResponseMessage{}
+	var err error
+
+	_, rsp.rawMsg, err = proto.ReadResp(reader)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(rsp.rawMsg) < 6 {
```

Comment:
Why `6`? Please add a comment

Reply:
It's the minimal required Kafka response length, we could define it as a `const`